README.md

```diff
@@ -410,3 +408,19 @@ The E2E tests use the `TeleporterMessenger` contract deployment transaction spec
 ```bash
 go generate ./...
 ```
+
+### Generate Protobuf Files
+
+[buf](github.com/bufbuild/buf/) is used to generate protobuf definitions for communication with the [Decider service](https://github.com/ava-labs/awm-relayer/blob/main/proto/decider/decider.proto). If you change any of the protobuf definitions you will have to regenerate the `.go` files. To generate these files, run the following command at the root of the project:
```

Comment:
Should we mention here that you need to manually pre-install `buf`? Or I wonder if we should modify the script so that it installs it for you, like we do with `abigen`.

Reply:
We should at minimum add this to the `Requirements` section in the top-level README

Reply:
Added this to the Requirements section. Installing the correct version of `buf` seems quite tricky and platform dependent, so I think sending people to the `buf` github page is the best course of action